platform/linux-generic/include/odp_schedule_scalable.h

```diff
@@ -89,6 +89,7 @@ typedef struct ODP_ALIGNED_CACHE {
 #define cons_type qschst_type
 #endif
 	odp_schedule_group_t sched_grp;
+	uint64_t loop_check[CONFIG_NUM_CPU];
```

Comment:
`ODP_THREAD_COUNT_MAX` should be used here.

Reply:
generally, ODP_THREAD_COUNT_MAX   is two bigger than CONFIG_NUM_CPU

Reply:
`ODP_THREAD_COUNT_MAX` doesn't refer directly to the number CPU threads. Instead, it is the number of ODP application threads (or processes), where `odp_init_local()` has been called. `ODP_THREAD_COUNT_MAX ` should be used with the `odp_thread_id()` API calls. By default, the values of `ODP_THREAD_COUNT_MAX` and `CONFIG_NUM_CPU` are the same.

Reply:
in our case
ODP_THREAD_COUNT_MAX  is  about 300
CONFIG_NUM_CPU is about 2-8 
very different

Reply:
There is a potential issue if multiple threads are running on the same core (allowed by API), since they end up sharing the same `loop_check`value but use a thread local `loop_cnt`. However, the potential value clash only causes a missed schedule round for low priority queues so I think we can live with this unless someone else has an objection. Calling schedule on multiple threads using the same core doesn't make much sense anyway.

Btw. `CONFIG_NUM_CPU` will be renamed to `CONFIG_NUM_CPU_IDS` in the head of master branch.